tests/integration-tests/tests/update/test_update.py

```diff
@@ -485,6 +485,45 @@ def _check_extra_json(command_executor, slurm_commands, host, expected_value):
     assert_that(result.stdout).is_equal_to('"{0}"'.format(expected_value))
 
 
+@pytest.mark.usefixtures("instance", "scheduler", "os")
+def test_update_rollback_change_instance_type(
```

Comment:
Is there another test where you can add this test? As in an existing test in which is also updating the cluster.

Reply:
I was able to combine it with test_update_instance_list because that test originally had one successful update and then another unsuccessful one because the commute fleet was not stop. I then took this test and then added on the portion where I stop the compute fleet and update, but with the rollback and change of instance type.